packages/static-module-record/test/benchmark-babel-plugin.js

```diff
@@ -0,0 +1,53 @@
+import Benchmark from 'benchmark';
```

Comment:
Not sure if we want this.

I needed something to check if I'm not introducing an awful performance regression. 

I'm not super happy with the benchmark. Differences between my implementation and previous implementation were less significant than differences between the same implementation executed at different times. 

I tried running both implementations in one suite and warming them up first. I could commit that too, but I doubt we want to keep multiple implementations laying around.

Example:

names correspond with fixtures, `2` means it's a new implementation (specifically one iteration before what's committed here)

```
small x 698 ops/sec ±11.71% (75 runs sampled)
large x 68.98 ops/sec ±3.19% (63 runs sampled)
exportheavy x 1,032 ops/sec ±4.57% (82 runs sampled)
small2 x 790 ops/sec ±2.57% (82 runs sampled)
large2 x 69.91 ops/sec ±5.07% (60 runs sampled)
exportheavy2 x 1,060 ops/sec ±2.10% (82 runs sampled)

small2 x 632 ops/sec ±11.21% (75 runs sampled)
large2 x 69.29 ops/sec ±2.80% (66 runs sampled)
exportheavy2 x 993 ops/sec ±3.53% (81 runs sampled)
small x 803 ops/sec ±5.18% (78 runs sampled)
large x 75.10 ops/sec ±2.25% (67 runs sampled)
exportheavy x 1,069 ops/sec ±2.15% (84 runs sampled)
```
With warmup:
```
warmup x 2,371 ops/sec ±9.89% (78 runs sampled)
small2 x 745 ops/sec ±4.39% (79 runs sampled)
large2 x 70.02 ops/sec ±3.31% (63 runs sampled)
exportheavy2 x 1,038 ops/sec ±2.53% (81 runs sampled)
small x 848 ops/sec ±3.36% (82 runs sampled)
large x 64.16 ops/sec ±8.13% (56 runs sampled)
exportheavy x 939 ops/sec ±6.42% (78 runs sampled)

warmup x 27.49 ops/sec ±7.32% (51 runs sampled)
small2 x 766 ops/sec ±2.59% (79 runs sampled)
large2 x 74.26 ops/sec ±2.04% (64 runs sampled)
exportheavy2 x 984 ops/sec ±4.15% (78 runs sampled)
small x 849 ops/sec ±3.78% (82 runs sampled)
large x 70.81 ops/sec ±5.13% (61 runs sampled)
exportheavy x 896 ops/sec ±9.47% (71 runs sampled)
```